trino-connector/integration-test/src/test/java/org/apache/gravitino/trino/connector/integration/test/TrinoQueryITBase.java

```diff
@@ -68,14 +68,17 @@ public class TrinoQueryITBase {
   protected static final String metalakeName = "test";
   protected static GravitinoMetalake metalake;
 
-  private static void setEnv() throws Exception {
+  private static AbstractIT abstractIT;
+
+  private void setEnv() throws Exception {
+    abstractIT = new AbstractIT();
```

Comment:
Here as well, it is strange to create an instance named "abstractXX".

Reply:
`TrinoQueryITBase` needs to start Gravitino server but it's not the subclass of `AbstractIT`, so we'd need to create a new instance. 

Why does it work before then?

The method `startIntegrationTest` before hand is a static method, so we can use `AbstractIT#startIntegrationTest`
directly.

Reply:
Is it possible for us to standardize the usage of `AbstractIT` class? Just like its name, we set it as an abstract class that requires inheritance to use.